plugins/plugin-chart-table/src/DataTable/DataTable.tsx

```diff
@@ -73,17 +72,17 @@ export default function DataTable<D extends object>({
   tableClassName,
   columns,
   data,
-  currentPage = 0,
+  ownCurrentState,
```

Comment:
Do you really need to pass `ownCurrentState` on to `DataTable`?

Reply:
fixed